### PR TITLE
Add forceLowercase preprocessor rule

### PR DESCRIPTION
**Pull request by basilisk487**

### CODE REVIEW

File: proxy/src/main/java/com/wavefront/agent/preprocessor/ReportPointForceLowercaseTransformer.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import wavefront.report.ReportPoint;
+
+/**
+ * Force lowercase transformer. Converts a specified component of a point (metric name, source name or a point tag
+ * value, depending on "scope" parameter) to lower case to enforce consistency
+ *
+ * Created by Vasily on 12/14/17.
```
Comment (panghy) [inline proxy/src/main/java/com/wavefront/agent/preprocessor/ReportPointForceLowercaseTransformer.java]: @author

File: proxy/src/main/java/com/wavefront/agent/preprocessor/ReportPointForceLowercaseTransformer.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+import java.util.regex.Pattern;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import wavefront.report.ReportPoint;
+
+/**
+ * Force lowercase transformer. Converts a specified component of a point (metric name, source name or a point tag
+ * value, depending on "scope" parameter) to lower case to enforce consistency
```
Comment (panghy) [inline proxy/src/main/java/com/wavefront/agent/preprocessor/ReportPointForceLowercaseTransformer.java]: period at end of sentence

**Comment by basilisk487** [comment]
@panghy Done

**Comment by panghy** [comment]
Can you try submitting the CR yourself? The fact that it was approved vs changes requested should mean that you can submit

**Pull request merged**